src/index.html

```diff
@@ -5,8 +5,333 @@
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="main">
+      <div class="card" data-qa="card">
+        <div class="card-top">
+          <a href="#" class="card-img">
+            <img src="./images/imac.jpeg"
+                 class="card-img__product"
+                 alt="APPLE A1419 iMac 27 retina 5k monoblock">
+          </a>
+        </div>
+        <div class="card-product">
+          <h2 class="card-product__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card-product__code">Product code: 195434</p>
+        </div>
+        <div class="stars">
+          <ul class="stars-quality stars-quality--4">
+            <li class="stars-quality__star"></li>
+            <li class="stars-quality__star"></li>
+            <li class="stars-quality__star"></li>
+            <li class="stars-quality__star"></li>
+            <li class="stars-quality__star"></li>
+          </ul>
+          <p class="stars__reviews">Reviews: 5</p>
+        </div>
+        <div class="card-count">
+          <p class="card-count__price">Price:</p>
+          <p class="card-count__value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card-button"
+          data-qa="card-hover"
+          >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card-top">
+          <a href="#" class="card-img">
+            <img src="./images/imac.jpeg"
+                 class="card-img__product"
+                 alt="APPLE A1419 iMac 27 retina 5k monoblock">
+          </a>
+        </div>
+        <div class="card-product">
+          <h2 class="card-product__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card-product__code">Product code: 195434</p>
+        </div>
+        <div class="stars">
```

Comment:
```suggestion
        </div>
        
        <div class="stars">
```

Add one empty line between sibling blocks inside the same parent block for both of them. Same for all similar cases